### DISCUSSION TITLE
How to properly add custom icon to shortcut?

### DISCUSSION THREAD

**Discussion by anderlli0053** [ScoopInstaller/Scoop#5067] [Help and Questions] [answered]
How to properly add custom icon to the scoop created program shortcut, saved in .ICO format and residing alongside other files? 

How to append it to the shortcut?

**Comment by lewis-yeung** [comment, marked as answer]
See [docs](https://github.com/ScoopInstaller/Scoop/wiki/App-Manifests). You can specify a custom icon for a shortcut via the 4th string of the shortcut entry. E.g.,

```json
"shortcuts": [
    [
        "target.exe",
        "Shortcut Name",
        "-arg1 /arg2",
        "icon.ico"
    ]
]
```

**Reply by anderlli0053** [reply]
> See [docs](https://github.com/ScoopInstaller/Scoop/wiki/App-Manifests). You can specify a custom icon for a shortcut via the 4th string of the shortcut entry. E.g.,
> 
> ```json
> "shortcuts": [
>     [
>         "target.exe",
>         "Shortcut Name",
>         "-arg1 /arg2",
>         "icon.ico"
>     ]
> ]
> ```

[Solved]

Thanks
